src/terminal7.ts

```diff
@@ -413,30 +413,28 @@ export class Terminal7 {
         const statusE = document.getElementById("peerbook-status") as HTMLSpanElement
         return new Promise((resolve, reject) => {
```

Comment:
Consider passing an `async` function to the Promise

Reply:
I'm afraid to change this so late in the game. I've added a TODO comment, hopefully we'll get to it sometime.